code/GrubsGame.cs

```diff
@@ -77,6 +87,11 @@ public override void ClientDisconnect( IClient client, NetworkDisconnectionReaso
 	{
 		GamemodeSystem.Instance?.OnClientDisconnect( client, reason );
 		UI.TextChat.AddInfoChatEntry( $"{client.Name} has left ({reason})" );
+
+		if ( client.Pawn is not Player player )
+			return;
+
+		PlayerColors[player.Color] = false;

```

Comment:
What happens if the player.Color is default in this case? Maybe we should check.